articles/language/expressions.md

```diff
@@ -600,6 +589,18 @@ the `C` operation will be invoked, and if it is false then only `D` will
 be invoked.
 This is similar to short-circuiting in other languages.
 
+## Copy-and-Update Expressions
+
+New arrays can be created from existing ones via copy-and-update expressions.
+A copy-and-update expression is an expression of the form `expression1 w/ expression2 <- expression3`, where `expression1` has to be of type `T[]` for some type `T`. The second `expression2` defines the indices of the element(s) to modify compared to the array in `expression1` and has to be either of type `Int` or of type `Range`. If `expression2` is of type `Int`, `expression3` has to be of type `T`. If `expression2` is of type `Range`, `expression3` has to be of type `T[]`.
```

Comment:
Should `` `T` `` be replaced by `` `'T` `` here to follow type parameter notation?

Reply:
well, is is a type parameter? I think we used that elsewhere this way.